components/input-number/style/index.less

```diff
@@ -73,7 +70,7 @@
     -moz-appearance: textfield;
     height: @input-height-base - 2px;
     transition: all 0.3s linear;
-    background-color: transparent;
+    background-color: @input-bg;
```

Comment:
https://github.com/ant-design/ant-design/commit/37ec04b21a662dd00fb7e729c6df7d53a082b5b1

Reply:
I'll have to look into this one a bit more.